Formula/neomutt.rb

```diff
@@ -35,6 +35,10 @@ class Neomutt < Formula
     depends_on "pkg-config" => :build
   end
 
+  # upstream build patch, https://github.com/neomutt/neomutt/pull/3370
+  # remove in next release
+  patch :DATA
```

Comment:
Looks like we need to do
```ruby
depends_on "ncurses"
```
since upstream do not support building with macOS `ncurses`.

Doing this should make the patch unnecessary.

Reply:
Cool, cool!

Reply:
Works for me.